src/mgr/ActivePyModules.cc

```diff
@@ -986,6 +987,15 @@ PyObject* ActivePyModules::get_perf_schema_python(
   return f.get();
 }
 
+PyObject* ActivePyModules::get_rocksdb_version()
+{
+  PyFormatter f;
+  f.dump_unsigned("major", ROCKSDB_MAJOR);
+  f.dump_unsigned("minor", ROCKSDB_MINOR);
+  f.dump_unsigned("patch", ROCKSDB_PATCH);
```

Comment:
Hi @tchaikov, thanks for your response! I like the idea of returning a number rather than a dict. However, I'm not quite sure I understand the logic/motivation behind this implementation.

I tried this patch myself, and the number `983040` was recorded in the Telemetry report. I understand that this is the encoded result of shifting `15`, the RocksDB minor version number, 16 bits to the left. If that's correct, where do the other RocksDB version numbers (major, patch) come into play? It seems that they are dropped by the | operator. What is the reasoning behind this?

Perhaps it's just that this is a method of encoding that I'm unfamiliar with.

Reply:
@ljflores fixed. see also https://github.com/facebook/rocksdb/blob/8d615a2b1d6d0784bf7efd3107284153f5378490/java/rocksjni/rocksjni.cc#L3702-L3706

Reply:
Cool. Fixed!